go_installer.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+work_dir=$(echo $1 | sed 's:/*$::')
+version="1.21.8"
```

Comment:
Can this instead be passed as an argument to the script so we can limit the number of places the Go Version needs to be changed?

Reply:
done